src/main/java/core/basesyntax/RandomFigure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class RandomFigure implements AreaCalculator, Draw {
```

Comment:
figure has no random features itself, no need to rename
```suggestion
public abstract class Figure implements AreaCalculator, Draw {
```